src/frontend/src/pages/CalendarPage/CalendarComponents/CalendarDayCard.tsx

```diff
@@ -31,13 +30,15 @@ interface CalendarDayCardProps {
 const CalendarDayCard: React.FC<CalendarDayCardProps> = ({ cardDate, events, teamTypes }) => {
   const [isCreateModalOpen, setIsCreateModalOpen] = useState(false);
 
+  const CardClick = () => (
```

Comment:
Great work! One thing - would you be able to make the calendar dates display some sort of shadow when they are hovered over? That way, it will be more clear to the user that they are able to click on the days.